easybuild/easyconfigs/g/gettext/gettext-0.19.8.1-GCCcore-6.4.0-Python-2.7.13-bare.eb

```diff
@@ -0,0 +1,42 @@
+easyblock = 'ConfigureMake'
+
+name = 'gettext'
+version = '0.19.8.1'
+pyver = '2.7.13'
+pysubver = '-bare'
+versionsuffix = '-Python-%s%s' % (pyver, pysubver)
+
+homepage = 'http://www.gnu.org/software/gettext/'
+
+description = """
+ GNU 'gettext' is an important step for the GNU Translation Project, as it
+ is an asset on which we may build many other steps. This package offers
+ to programmers, translators, and even users, a well integrated set of tools
+ and documentation
+"""
+
+toolchain = {'name': 'GCCcore', 'version': '6.4.0'}
+
+source_urls = [GNU_SOURCE]
+sources = [SOURCE_TAR_GZ]
+checksums = ['ff942af0e438ced4a8b0ea4b0b6e0d6d657157c5e2364de57baa279c1c125c43']
+
+builddependencies = [
+    ('binutils', '2.28'),
+    ('Python', pyver, pysubver),
```

Comment:
@JackPerdue Can you clarify why you are including Python as a (build) dep here?

Is it just because `python` is used during the build/install procedure of `gettext`?

At first sight, none of the installed files require `python` at runtime, i.e. there are no Python modules or scripts installed with `gettext`?

Reply:
OK, I think I know what's going on...

The `libxml2` dependency included here was built with Python bindings enabled (because `Python` was included as a build dep), and so you're loading `Python` as a build dep here as well because you assume that it's required by `libxml2` as a runtime dep, am I right?

I doubt that's the case though: although `libxml2` provides Python bindings, I don't see why they would be used by `gettext`, since it doesn't provide anything that requires a `Python` runtime?

The `libxml2` with and without `Python` bindings is a bit of a long-standing mess.

Maybe we should try and get rid of it by installing `libxml2` without Python bindings, and then adding an additional `libxml2-python` which *only* provides the Python bindings (and includes both `libxml2` and `Python` as dependencies)?

Reply:
`gettext` on top of `libxml2` without Python bindings tackled in #5082